src/frontend-samples/clashdetection-sample/ClashDetectionApis.ts

```diff
@@ -0,0 +1,194 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { assert } from "@bentley/bentleyjs-core";
+import { IncludePrefix, request, Response } from "@bentley/itwin-client";
+import { IModelApp } from "@bentley/imodeljs-frontend";
+import { IModelHubClient, VersionQuery } from "@bentley/imodelhub-client";
+import { NoSignInIAuthClient } from "NoSignInIAuthClient";
+import ClashDetectionApp from "./ClashDetectionApp";
+
+// NOTE: The following samples for calling the Validation APIs will not succeed within the
```

Comment:
This general comment doesn't apply anymore, right?

Maybe add specific comments on each of the methods that cannot be run in the showcase?  Ex. This method will not succeed within the sample showcase because the user token does not have sufficient access to <whatever>.

Reply:
Good point!